tests/integration/schema_registry/test_api.py

```diff
@@ -0,0 +1,257 @@
+from uuid import uuid1
+
+from confluent_kafka.schema_registry import CompatibilityType, SchemaRegistryClient
+from confluent_kafka.schema_registry.config import SchemaRegistryConfig
+
+
+def _subject_name(prefix):
+    return "-".join([prefix, str(uuid1())])
+
+
+def test_api_get_schema(kafka_cluster, load_avsc):
+    """
+    Registers a schema then retrieves it using the schema id returned from the
+    call to register the Schema.
+
+    Args:
+        kafka_cluster (KafkaClusterFixture): Kafka Cluster fixture
+        load_avsc (callable(str)): Schema fixture constructor
+
+    Raises:
+        AssertionError if the Schema retrieved by ID does not match the expected
+        Schema
+    """
+    sr_conf = SchemaRegistryConfig(kafka_cluster.schema_registry())
+    sr = SchemaRegistryClient(sr_conf)
+
+    schema = load_avsc('basic_schema.avsc')
+
+    version = sr.register_schema(str(uuid1()), schema)
+    schema2 = sr.get_schema(version.schema_id)
+
+    assert schema2 == schema
+
+
+def test_api_get_subjects(kafka_cluster, load_avsc):
+    """
+    Populates KafkaClusterFixture SR instance with a fixed number of subjects
+    then verifies the response includes them all.
+
+    Args:
+        kafka_cluster (KafkaClusterFixture): Kafka Cluster fixture
+        load_avsc (callable(str)): Schema fixture constructor
+
+    Raises:
+        AssertionError if all registered subjects are not returned
+    """
+    sr_conf = SchemaRegistryConfig(kafka_cluster.schema_registry())
+    sr = SchemaRegistryClient(sr_conf)
+
+    avscs = ['basic_schema.avsc', 'primitive_string.avsc',
+             'primitive_bool.avsc', 'primitive_float.avsc']
+
+    for avsc in avscs:
+        sr.register_schema(_subject_name(avsc), load_avsc(avsc))
+
+    registered = sr.get_subjects()
+    assert len(registered) >= 4
```

Comment:
This does not verify that these 4 schemas were registered.
Suggest using a unique subject name for each spin and verify they're seen in the list.